src/test/java/uk/gov/pay/directdebit/gatewayaccounts/services/GatewayAccountServiceTest.java

```diff
@@ -57,13 +56,13 @@
     private MandateFixture mandateFixture = MandateFixture.aMandateFixture()
             .withGatewayAccountFixture(gatewayAccountFixture);
     @Mock
-    private GatewayAccountDao mockedGatewayAccountDao;
+    private GatewayAccountDao gatewayAccountDao;
```

Comment:
It’s a matter of taste but I actually prefer having a prefix on mock objects (though I prefer “mock” to “mocked”) because it makes it really clear we’re dealing with a mock object

Reply:
I guess I was finding it really obvious it was a mock as mentions of gatewayAccountDao are usually preceded by "when"s, for e.g. `when(gatewayAccountDao.findByExternalId(accountId)).thenReturn`...

Reply:
I appreciate that. Though I do like how in a line like `when(mockGatewayAccountDao.findByExternalid(accountId).thenReturn(mockGatewayAccount)` the prefix makes it clear which variables are mocks and which aren’t. I think a prefix is particularly useful when _writing_ (rather than reading) code because I sometimes lose track of what’s a mock and what isn’t.

I think we should probably have a team standard for this but we don’t have one right now, so I’m happy to go with what you prefer in this PR.